contracts/src/state/CommitmentAccumulator.sol

```diff
@@ -85,11 +85,6 @@ contract CommitmentAccumulator is ICommitmentAccumulator {
         internal
         view
     {
-        // Check length.
-        if (path.length != _merkleTree.depth()) {
-            revert InvalidPathLength({expected: _merkleTree.depth(), actual: path.length});
-        }
-
```

Comment:
Or should we rather check that `path.length <= _merkleTree.depth()`? I think this would be right